lib/AbstractHandle/Utils/MongoUtil.py

```diff
@@ -81,10 +81,11 @@ def get_hid_counter(self):
         """
         get current handle id counter
         """
-        counter = self.hid_counter_collection.find({'_id': {'$eq': self._HID_COUNTER_ID}})
+        cursor = self.hid_counter_collection.find({'_id': {'$eq': self._HID_COUNTER_ID}})
+        counter = list(cursor)
 
-        if counter.count():
-            return counter.next().get('hid_counter')
+        if len(counter):
+            return counter[0].get('hid_counter')
```

Comment:
It seems like this function is only used in tests: https://github.com/search?q=repo%3Akbase/handle_service2%20get_hid_counter&type=code

@Tianhao-Gu can it be moved to a test helper?

Reply:
yea. if it's only used in tests, we can move it to a test helper.

Reply:
👍

Reply:
:+1: